src/mpl/src/MacroPlacer.cpp

```diff
@@ -815,6 +815,7 @@ void MacroPlacer::findMacros()
   }
 
   logger_->info(MPL, 5, "Found {} macros.", macros_.size());
+  logger_->metric("floorplan__design__instance__count__macros", macros_.size());
```

Comment:
These metrics should be extracted from OpenDB  instead of from tool data structures. I am refactoring the area report that extract all of the count/area/util metrics (much like the GUI display control) which we will use for all metrics reporting